data_prep/introspector.py

```diff
@@ -469,6 +488,10 @@ def _parse_arguments() -> argparse.Namespace:
                       type=str,
                       default='',
                       help='Output directory.')
+  parser.add_argument('-l',
+                      '--local',
```

Comment:
Could we generalize this as `--endpoint`, with a default value of introspector.oss-fuzz.com/api ?

Reply:
Yeah, done